jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/SpringBootConfiguration.java

```diff
@@ -42,7 +42,8 @@ public static SpringBootConfiguration from(JavaProject project) {
     final Properties properties = SpringBootUtil.getSpringBootApplicationProperties(
       SpringBootUtil.getSpringBootActiveProfile(project),
       JKubeProjectUtil.getClassLoader(project));
-    final int majorVersion = SpringBootUtil.getSpringBootVersion(project)
+    Optional<String> springBootVersion = SpringBootUtil.getSpringBootVersion(project);
```

Comment:
Few tests have started failing after this change:

```
[ERROR] Errors: 
[ERROR]   SpringBootGeneratorTest.customize_whenGeneratorModeWatchAndDevtoolsSecretPresent_thenAddDevtoolsSecretEnvVar:220 » NoSuchElement No value present
[ERROR]   SpringBootGeneratorTest.customize_withEmptyList_shouldReturnAddedImage:141 » NoSuchElement No value present
[ERROR]   SpringBootWatcherTest.getPortForwardUrl:102 » NoSuchElement No value present
[INFO] 

```

Reply:
Fixed